app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java

```diff
@@ -251,6 +253,15 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
                 }
             });
         }
+
+        _verticalScrollView.getViewTreeObserver().addOnGlobalLayoutListener(() -> {
+            _hlEditor.post(() -> {
```

Comment:
As the HighlightingEditor is now under a linearlayout, this imitates fillViewport